internal/js_parser/global_name_parser.go

```diff
@@ -18,10 +18,24 @@ func ParseGlobalName(log logger.Log, source logger.Source) (result []string, ok
 	}()
 
 	lexer := js_lexer.NewLexerGlobalName(log, source)
-
-	// Start off with an identifier
-	result = append(result, lexer.Identifier.String)
-	lexer.Expect(js_lexer.TIdentifier)
+	// Start off with an identifier or `import`
+	if allowImportMeta {
+		if lexer.Token == js_lexer.TImport {
+			result = append(result, lexer.Identifier.String)
+			lexer.Next()
+			lexer.Expect(js_lexer.TDot)
+			result = append(result, lexer.Identifier.String)
+			lexer.Expect(js_lexer.TIdentifier)
```

Comment:
I didn't strictly limit to `import.meta` here. (`import.foo` is allowed and `import['meta']` is not allowed)